guides/v2.3/graphql/reference/customer.md

```diff
@@ -117,3 +121,417 @@ The following call returns information about the logged-in customer.
+
+#### Example usage
+
+The following call creates an address for the specified customer.
+
+**Request**
+
+``` text
+mutation {
+  createCustomerAddress(input: {
+    region: {
+        region: "Arizona"
+        region_id: 4
+        region_code: "AZ"
+    }
+    country_id: US
+    street: ["123 Main Street"]
+    telephone: "7777777777"
+    postcode: "77777"
+    city: "Phoenix"
+    firstname: "Bob"
+    lastname: "Loblaw"
+    default_shipping: true
+    default_billing: false
+  }) {
+    id
+    customer_id
+    region {
+      region
+      region_id
+      region_code
+    }
+    country_id
+    street
+    telephone
+    postcode
+    city
+    default_shipping
+    default_billing
+  }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "createCustomerAddress": {
+      "id": 4,
+      "customer_id": 5,
+      "region": {
+        "region": "Arizona",
+        "region_id": 4,
+        "region_code": "AZ"
+      },
+      "country_id": "US",
+      "street": [
+        "123 Main Street"
+      ],
+      "telephone": "7777777777",
+      "postcode": "77777",
+      "city": "Phoenix",
+      "default_shipping": true,
+      "default_billing": false
+    }
+  }
+}
+```
+
+### Update customer address
+
+Updates the customer's address.
+
+#### Syntax
+
+`mutation: updateCustomerAddress`
+
+#### Example usage
+
+The following call updates the customer's city and postcode.
+
+**Request**
+
+``` text
+mutation {
+  updateCustomerAddress(id:3, input: {
+    city: "New City"
+    postcode: "5555"
+  }) {
+    id
+  }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "updateCustomerAddress": {
+      "id": 3
+    }
+  }
+}
+```
+### Delete customer address
+
+Uses the customer's address `id` to delete the address.
+
+#### Syntax
+
+`mutation: deleteCustomerAddress`
+
+#### Example usage
+
+The following call deletes a customer's address.
+
+**Request**
+
+``` text
+mutation {
+  deleteCustomerAddress(id: 4)
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "deleteCustomerAddress": true
+  }
+}
+```
+
+### Manage customer tokens
+{:.no_toc}
+
+Use these mutations to create or revoke a customer's token.
+
+**Manage customer tokens attributes**
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`email` | String | The customer's email address
+`password` | String | The customer's password
+
+### Generate a customer token
+
+Creates a new customer token.
+
+#### Syntax
+
+`mutation: generateCustomerToken`
+
+#### Example usage
+
+The following call creates a new customer token.
+
+**Request**
+
+``` text
+mutation {
+	generateCustomerToken(
+        email: "bobloblaw@example.com"
+        password: "b0bl0bl@w"
+    ) {
+        token
+    }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "generateCustomerToken": {
+      "token": "ar4116zozoagxty1xjn4lj13kim36r6x"
+    }
+  }
+}
+```
+
+### Revoke a customer token
+
+Revokes the customer's token.
+
+#### Syntax
+
+`mutation: remokeCustomerToken`
```

Comment:
Typo. Also, as usual, use the same format shown in the Graphiql browser

Reply:
fixed